apps/designer/app/designer/features/sidebar-left/panels/components/thumbnail.tsx

```diff
@@ -0,0 +1,135 @@
+import { useEffect, useRef, useState } from "react";
+import {
+  Box,
+  Button,
+  Flex,
+  ProgressBar,
+  Tooltip,
+} from "~/shared/design-system";
+import { useClickAway, useInterval } from "react-use";
+import placeholderImage from "~/shared/images/image-placeholder.svg";
+import brokenImage from "~/shared/images/broken-image-placeholder.svg";
+import { useSubmit } from "@remix-run/react";
+import { Asset } from "~/designer/features/sidebar-left/types";
+import { Cross2Icon } from "@radix-ui/react-icons";
+import { useHotkeys } from "react-hotkeys-hook";
+import { shortcuts } from "~/shared/shortcuts";
+
+const useImageWithFallback = ({ path }: { path: string }) => {
+  const [src, setSrc] = useState(placeholderImage);
+
+  useEffect(() => {
+    const newImage = new Image();
+    newImage.onload = () => setSrc(path);
+    newImage.onerror = () => setSrc(brokenImage);
+    newImage.src = path;
+  }, [path]);
+
+  return src;
+};
+
+export const AssetManagerThumbnail = ({
+  path,
+  alt,
+  status,
+  name,
+  id,
+}: Asset) => {
+  const submit = useSubmit();
+  const isUploading = status === "uploading";
+  const src = useImageWithFallback({ path });
+  const [isToolTipOpen, setTolltipOpen] = useState(false);
+  const [isDeleting, setIsDeleting] = useState(false);
+  const [progressBarPercentage, setProgressBarPercentage] = useState(0);
+
+  const closeTooltip = () => setTolltipOpen(false);
+
+  const tooltipRef = useRef(null);
+  useClickAway(tooltipRef, closeTooltip);
+  useHotkeys(shortcuts["escape"], closeTooltip);
```

Comment:
radix tooltip is supposed to handle both, esc and click outside https://www.radix-ui.com/docs/primitives/components/tooltip#content

Reply:
I see what happened, it had to be updated. Updated it to v1 and it now supports everything